addons/web_tour/static/src/js/running_tour_action_helper.js

```diff
@@ -116,11 +116,20 @@ var RunningTourActionHelper = core.Class.extend({
     },
     _drag_and_drop: function (values, to) {
         var $to;
-        var elementCenter = values.
-        $element.offset();
```

Comment:
:grimacing:

Reply:
@rdeodoo  Note about this whole thing that I would rather have the whole step definition run through those methods (or maybe a `runParam` attribute ?) so that we could simply use fields with additional info instead of having to build a silly string format. But that sounded like a big general change for such a specific need, so I went for the string-based approach.

Reply:
The smiley was just for that weird looking line that received  line return probably during a regex replace :)
About what you say, I'd juste create a new whole event / method for that as stated in https://github.com/odoo/odoo/pull/85778#discussion_r850755379
What do you think?

Reply:
Yes, I agree about the split - I'm on it. I was just stating this "for the record".